### DISCUSSION TITLE
For which kinds of data is `ENCRYPTION_KEY` used?

### DISCUSSION THREAD

**Discussion by felix-sanchez-unmind** [langfuse/langfuse#3972] [Support] [answered]
In the documentation , it says that ENCRYPTION_KEY is used to encrypt sensitive data. I was wondering what counts at sensitive data. I'm particularly interested on how traces are encrypted at rest, since in our company we are dealing with sensitive conversations. Thanks!

**Comment by marcklingen** [comment, marked as answer]
Thanks for the question. This was a great nudge to improve the documentation with regards to encryption: https://langfuse.com/docs/deployment/self-host#encryption

Let me know in case you have any follow-up questions!
